### PR TITLE
:bug: Drop sudo modification from ubuntu flavors

### PR DESCRIPTION
**Pull request by Itxaka**
We were "fixing" the sudo command but that resulted in sudo not working on livecd and weird behaviour.

Leave it as it is as it should come with proper perms from the package directly

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: images/Dockerfile.ubuntu [hunk 1]
```diff
@@ -243,8 +243,5 @@ RUN ln -s /usr/bin/grub-editenv /usr/bin/grub2-editenv
 RUN systemctl enable systemd-networkd
 RUN systemctl enable ssh
 
-# Fixup sudo perms
-RUN chown root:root /usr/bin/sudo && chmod 4755 /usr/bin/sudo
-
```
[the remark this answers is not in the document]
Comment (Itxaka) [reply images/Dockerfile.ubuntu:248]: I just think that we should not change sudo perms just becuase without a note. sudo seems to be ok out of the box in ubuntu so either this was an issue when the flavor was first introduced and no longer valid or was a workaround for something that we dont know.

@mudler will check when he is back, see if he remembers why it was introduced.

**Review by jimmykarily — DISMISSED** [review]

File: images/Dockerfile.ubuntu (hunk 1 above)
[the remark this answers is not in the document]
Comment (mudler) [reply images/Dockerfile.ubuntu:248]: well, back then that was a huge issue as we could not use sudo inside a kairos box. did you check permissions on a running system?

[the remark this answers is not in the document]
Comment (Itxaka) [reply images/Dockerfile.ubuntu:248]: Well, I mean sudo works out of the box so I dont know what Im supposed to test here xD

**Review by mauromorales — APPROVED** [review]
lgtm

File: images/Dockerfile.ubuntu (hunk 1 above)
[the remark this answers is not in the document]
Comment (Itxaka) [reply images/Dockerfile.ubuntu:246]: @mudler any more comment here? What Im supposed to check? sudo works out fo the box with this patch merged so not sure what was the failure

**Comment by mauromorales** [comment]
@Itxaka should we close this one?

**Comment by Itxaka** [comment]
Probably yeah

**Pull request closed without merging**